packages/api/cms-api/src/blocks/blocks.module.ts

```diff
@@ -8,13 +8,13 @@ import { BlocksTransformerService } from "./blocks-transformer.service";
 import { CommandsService } from "./commands.service";
 import { DiscoverService } from "./discover.service";
 
-export interface BlocksModuleOptions {
+export interface BlocksModuleFactoryOptions {
```

Comment:
I'm not sure about the naming. Maybe it should be `BlocksModuleOptions` for the actual options, and `BlocksModuleSyncOptions` for `forRoot`, and `BlocksModuleAsyncOptions` for `forRootAsync`. At least that's how MikroORM does it: https://github.com/mikro-orm/nestjs/blob/master/src/mikro-orm.module.ts#L20

This isn't about this PR in particular, rather a general concern.

Reply:
:+1:

Reply:
for most other modules I removed one of the two interfaces, for this one I made a mistake...

Reply:
I think it's interesting that the linter didn't complain about it either